src/components/action-button.js

```diff
@@ -10,9 +10,8 @@ export default function ActionButton({ layout }) {
       color = component.fontColor;
     }
   }
-  const button = document.createElement('button');
   button.setAttribute('style', `color: ${color}; 'font-size': ${fontSize}`);
-  button.appendChild(document.createTextNode('Not React'));
+  button.innerHTML = 'Not React';
```

Comment:
should avoid using innerHTML. It's not an issue here, but as we want to insert an user inputted expression, it can get nasty if not escaped correctly. There is a comment on it here: https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML